lib/knapsack/adapters/spinach_adapter.rb

```diff
@@ -0,0 +1,42 @@
+require 'spinach'
```

Comment:
@ayufan I found out this causes problems when someone don't use spinach. I did fix in knapsack 1.11.1
https://github.com/ArturT/knapsack/blob/master/CHANGELOG.md#1111